### ISSUE TITLE
Manual anchors should be generated from text, not from XSL generated ID

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#13264]
The manual generates anchors on all headings from the XSL `generate-id` function:

```xml
    <xsl:template match="h3" mode="content">
        <xsl:variable name="id" select="generate-id(.)"/>

        <h2 id="{$id}">
            <a href="#{$id}" name="{$id}">
                <xsl:apply-templates mode="content"/>
            </a>
        </h2>
    </xsl:template>
```

This could look like this:
https://www.jooq.org/doc/dev/manual/sql-building/sql-statements/select-statement/#NAAB07

Those IDs are unique within a document, but probably not stable between executions or even between versions of the manual. As such, people will likely create broken links, e.g. on Stack Overflow.

This is an even bigger problem for the single page manual, where the headings contained in the TOC produce the title element, but the lower level headings are completely meaningless.

**Comment by lukaseder** [comment]
The single page manual doesn't generate those anchors

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.17.0
